data/resources.go

```diff
@@ -70,3 +72,16 @@ func populateItems() (items []models.Resource, err error) {
 
 	return items, nil
 }
+
+func filterItems(items []models.Resource, options Options) []models.Resource {
```

Comment:
Maybe a comment here on what the function does

Reply:
fair cop